pydoctor/epydoc2stan.py

```diff
@@ -316,13 +316,14 @@ def from_ast_annotations(cls, obj: model.Documentable, annotations: Mapping[str,
         linker = _EpydocLinker(obj)
         annotations = {name: AnnotationDocstring(value).to_stan(linker)
                        for name, value in annotations.items()}
-        return_type = FieldDesc()
-        try:
-            return_type.body = annotations.pop("return")
-        except KeyError:
-            pass
         ret_value = cls(obj, annotations)
```

Comment:
Moving the `pop()` call below the constructor call means that the annotations can now include a `return` key while that wasn't possible in the original code. I don't think this will cause problems, but it is confusing and should be fixed.

I'm wondering though: if having a pseudo-argument named `return` is useful for annotations, is it also useful for epytext fields?